Classes/Task/IndexQueueWorkerTaskAdditionalFieldProvider.php

```diff
@@ -56,42 +56,43 @@ public function getAdditionalFields(
     ) {
         $additionalFields = [];
 
-        $this->isTaskInstanceofIndexQueueWorkerTask($task);
-
-        if ($schedulerModule->CMD == 'add') {
-            $taskInfo['site'] = null;
-            $taskInfo['documentsToIndexLimit'] = 50;
-            $taskInfo['forcedWebRoot'] = '';
-        }
-
-        if ($schedulerModule->CMD == 'edit') {
-            $taskInfo['site'] = $task->getSite();
-            $taskInfo['documentsToIndexLimit'] = $task->getDocumentsToIndexLimit();
-            $taskInfo['forcedWebRoot'] = $task->getForcedWebRoot();
+        if ($this->isTaskInstanceofIndexQueueWorkerTask($task)) {
```

Comment:
I would propose to keep the usages of isTaskInstanceofIndexQueueWorkerTask as it was because otherwise we just introduce another level maybe it is an option to rename isTaskInstanceofIndexQueueWorkerTask in checkIsTaskInstanceofIndexQueueWorkerTask ?